@here/harp-mapview/lib/geometry/TileGeometryLoader.ts

```diff
@@ -25,6 +25,21 @@ import { PerformanceStatistics } from "../Statistics";
 import { Tile } from "../Tile";
 import { TileGeometryCreator } from "./TileGeometryCreator";
 
+function addDiscardedTileToStats(tile: Tile) {
```

Comment:
this should move to Tile.ts

Reply:
This must be done on every call to TileGeometryLoader.cancel(). The simplest way to do it is to do it inside the cancel function. Otherwise, we need to create a new function in tile to ensure that we call TileGeometryLoader.cancel() and log the stats.
Furthermore, the rest of the stats are also logged from TileGeometryLoader.

Reply:
thanks for clarifiing, I did not look at the bigger picture there, that the other stats are also there.